### ISSUE TITLE
Re-enable SDL validation in release/3.1

### ISSUE THREAD

**Issue by wtgodbe** [dotnet/winforms#2176]
SDL Validation is still failing in the release/3.1 internal builds. We should figure out why, then re-enable it (once the branch opens back up after preview2 is finished). 

See https://github.com/dotnet/winforms/pull/2175, https://github.com/dotnet/arcade/issues/4099.

CC @AdamYoblick @RussKie @riarenas @JohnTortugo

**Comment by JohnTortugo** [comment]
Is that branch updated with latest changes in Arcade?

/cc @sunandabalu

**Comment by AdamYoblick** [comment]
Sorry, this slipped through the cracks for me. Did this get resolved?

**Comment by riarenas** [comment]
Sorry, I missed this one yesterday,

This could be due to the area path changing underneath us again? @wtgodbe do you have an example build where you tried to re-enable it to see the failures?

**Comment by AdamYoblick** [comment]
Yeah Area path was an issue for us on the dev div end as well. I'm updating the path in our master branches. I assume I'll need to do the same in the release branches as well?

**Comment by riarenas** [comment]
@sunandabalu has the guidance on how we're trying to get unblocked, and what the new area path should be.

**Comment by wtgodbe** [comment]
Here's a failing build: https://dnceng.visualstudio.com/internal/_build/results?buildId=399756

We can put up PRs for this now if the fix is known, but please don't merge into 3.1 yet, as that branch is locked down for the preview2 build.

**Comment by riarenas** [comment]
Yeah, that's the area path error: 

```
TSA was given an invalid Area Path: DevDiv\\NET Core\\Winforms\r\n
```

@sunandabalu is talking with the devdiv folks to try and understand what the final area path is going to be.

**Comment by AdamYoblick** [comment]
New area path for winforms are as follows (from what I can see):

DevDiv\NET Framework\Winforms\Designer\Core
DevDiv\NET Framework\Winforms\Runtime\Core

**Comment by AdamYoblick** [comment]
These are the paths I've changed to in the PR's into master for us.

**Comment by RussKie** [comment]
Is there anything outstanding, or can we close this?

**Comment by wtgodbe** [comment]
SDL is still disabled in 3.1 - I can look into fixing up the Area Path tomorrow morning, but we'll have to wait on merging the fix until the branches open again for 3.1.3.

**Comment by RussKie** [comment]
Thank you. No rush, it's been this long without any fires...
Do we need to do anything for the master?

**Comment by elachlan** [comment]
@merriemcgaw the team should probably review this to see if it needs to be closed.

**Comment by merriemcgaw** [comment]
Yeah, this was specific to 3.1 and no longer a concern. Thanks for flagging!
